### ISSUE TITLE
Sample1.html renders as all disabled in RAD

### ISSUE THREAD

**Issue by aerwin** [maqetta/maqetta#2940]
Making high to investigate... but not sure we're really doing anything wrong in Maqetta... suspecting RAD design previewer might not be dealing well with our state code...

After importing a Maqetta project into RAD, if Sample1.html is opened in the RAD page editor most of the widgets are disabled:

![screenshot](http://i.imgur.com/RtZTY.png)

However, when previewing from RAD (e.g., launching in a browser), it renders fine:

![screenshot](http://i.imgur.com/PZB80.png)

**Comment by aerwin** [comment]
So, I've determined there's no real problem in Maqetta code. It seems that the RAD page editor is interpreting an atrribute setting like `disabled="false"` to mean to render the item as disabled.

I've alerted the RAD team to the problem. In addition, with commit fe972f48155be1c9ac4ca3a54f5f67e0b4dfbead , I've removed extraneous attributes like that from HTML elements samples so they work in RAD "out of the box" even if RAD doesn't fix their bug.
